modules/shaders.js

```diff
@@ -19,24 +19,42 @@ class CLCustomWaveIlluminationShader extends StandardIlluminationShader {
     const float MAX_INTENSITY = 1.1;
     const float MIN_INTENSITY = 0.8;
 
+    // Normalized wave function to create the pulsating rings
```

Comment:
This is awesome, I can't thank you enough for doing this. I hope it will allow other creators to dip their toe into shaders